TMBuilder/src/main/java/com/ebay/tool/thinmodelgen/gui/menu/TMBuilderMenu.java

```diff
@@ -1087,7 +1082,7 @@ public String getDescription() {
 
   }
 
-  class JavaFileFilter extends FileFilter {
+  class JavaKotlinFileFilter extends FileFilter {
```

Comment:
I see, you encapsulated both in this single file filter. Please rename this to AndroidFileFilter so it isn't name tied to a single file format.

Reply:
Got it, Changed it to AndroidFileFilter